### PR TITLE
Show UI for displaying university details

### PR DESCRIPTION
**Pull request by Avanti0532**
The user interface for displaying university details has been set up. The page will display below information about the universities.
University name
University description
Rank
Acceptance rate
Tuition
University Type
Location
Weather
University website link

## Issue Related
#88 

## Type of change
Added Show UI for universities

Please delete options that are not relevant.
- [x] New feature (non-breaking change which adds functionality)

## Code coverage
94.16%

## Screenshot
show

### CODE REVIEW

File: app/views/universities/show.html.erb [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+</div>
+
+<div class="container">
+
+  <section class="home_banner_area_university">
+    <div class="banner_inner d-flex align-items-center">
+      <div class="overlay bg-parallax" data-stellar-ratio="0.9" data-stellar-vertical-offset="0" data-background="" style="transform: translateY(-23.5182px);"></div>
+      <div class="container">
+        <div class="col-lg-12">
+          <h1 style="color: white; font-size: 40px; font-family: 'Karla', sans-serif; font-weight: bold"><%= @university.university_name %></h1>
```
Comment (hpitawela) [inline app/views/universities/show.html.erb]: I think it is good if we reduce inline styling and make a separate university.css.

Comment (hjmjohnson) [reply app/views/universities/show.html.erb]: I agree with @hpitawela   inline styling can be a maintenance nightmare.

**Comment by hjmjohnson** [comment]
@Avanti0532 @linhpha @juliachaloupka @hpitawela  I'm a little concerned about the size of this PR.  It seems to interleave several commit from several people.  Each patch within a PR should be for single topical change.  

I'd recommend that your team spend a little time learning how to re-order patches or how tot recreate patches with git.  Keeping the contents of a PR clean and single topic will eventually greatly increase your productivity as a team.

```
git reset origin/master
git add -p 
git commit -m "Topic 1 stuff (i.e. documentation changes)"
git add -p
git commit -m"Topic 2 stuff (i.e. bug fix#1)"
git add -p
git commit -m"Topic 3 stuff (i.e. add new features)"

```

**Comment by linhpha** [comment]
> @Avanti0532 @linhpha @juliachaloupka @hpitawela I'm a little concerned about the size of this PR. It seems to interleave several commit from several people. Each patch within a PR should be for single topical change.
> 
> I'd recommend that your team spend a little time learning how to re-order patches or how tot recreate patches with git. Keeping the contents of a PR clean and single topic will eventually greatly increase your productivity as a team.
> 
> ```
> git reset origin/master
> git add -p 
> git commit -m "Topic 1 stuff (i.e. documentation changes)"
> git add -p
> git commit -m"Topic 2 stuff (i.e. bug fix#1)"
> git add -p
> git commit -m"Topic 3 stuff (i.e. add new features)"
> ```

@Avanti0532  and I were working together on this feature so that why this PR contains both my commits and her commits. That was not unintentional and the team is aware of that. All of the commits in this PR are related to creating the UI to show details about university and don't really touch on any other issue.  The amount of change is 72 lines, which we think is reasonable for the size of the change. It's true that our commit messages might not have been clear so it might make people confuse about what we are changing. We will close this PR and redo these commits with better message. We will also sit down and think about how we can collaborate in the same issue but don't let the commit history gets too large and make it difficult to understand.

**Pull request closed without merging**